cli/src/upload.rs

```diff
@@ -153,12 +162,11 @@ impl ContractDefinition {
             ))
         })?;
 
-        let wasm = doc["wasm"].as_str().ok_or_else(|| {
-            CliError::UserError(format!(
-                "Malformed contract definition file \"{}\": missing string field \"wasm\"",
-                filename
-            ))
-        })?;
+        let wasm = doc["wasm"]
+            .as_str()
+            .map(|y| {
```

Comment:
I think you can just call `.map(ToString::to_string)` here.

Reply:
fixed